x-pack/legacy/plugins/ml/public/data_frame/pages/transform_management/components/transform_list/common.ts

```diff
@@ -105,6 +105,17 @@ export interface DataFrameTransformListRow {
   stats: DataFrameTransformStats;
 }
 
+export interface DataFrameTransformEndpointData {
```

Comment:
this name isn't that obvious to me, perhaps something like `DataFrameTransformEndpointRequestData` would be clearer.
This would then match `DataFrameTransformEndpointResultData`.
The names are pretty long and I wonder if they could be shortened by dropping the `Endpoint` or `Data` parts

Reply:
Renamed to `DataFrameTransformEndpointRequest` and `DataFrameTransformEndpointResult` in f7a9d5de55631c0e03cb43586efbddc0b2a15555